src/modules/methods.js

```diff
@@ -0,0 +1,20 @@
+const postScore = async (user, score) => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/sNLypj2W3eNffYuzUA4N/scores', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
+    body: JSON.stringify({ user, score }),
+  });
+  const data = await response.json();
+  return data.result;
+};
+
+const getScore = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/sNLypj2W3eNffYuzUA4N/scores');
+  const data = await response.json();
+  return data.result;
+};
```

Comment:
Hello @Iqbal-Elham, you did a great job hitting the API and your project looks very well organized 😄. Although, please notice, as requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/209009119-1774add8-d6f5-4191-a4a2-7ff66f52aef1.png)

This means that the refresh button should _update_ your list of scores 😉. Currently, clicking on the refresh button renders all the elements again, causing this bug:
![image](https://user-images.githubusercontent.com/42799579/209009396-ffb2e5f6-72d2-477e-919b-2ca560b53e20.png)

To fix this and make your project work as expected, please make sure that your refresh button updates the list of scores, instead of adding new scores every time it is clicked 😉.

Cheers and keep coding 😄!